modules/guide/pages/xml-generation.adoc

```diff
@@ -134,7 +134,7 @@ TIP: Simply splitting the XPath string on `'/'` won't work. You can use regular
 NOTE: We know that extracting the `steps` from an XPath is not rocket science and that you can do it yourself; but it would have been much easier for everyone, if we had provided this information pre-processed and ready for you inside `fields.json`. This is something we intend to do in a future release of the SDK. 
 
 === Getting the element order right
-Ideally we would like you to be able to generate the XML without ever needing to look at the XSD. For the moment however, there is a piece of necessary information that you can only find inside the XSD: the correct order of XML elements within their parent element. This is because the order of XML elements inside an `xsd:sequence` is significant.
-
-NOTE: We are aware that having to open the XSD just to lookup the order of XML elements is not very convenient. So we plan to add this in `fields.json` in a future release of the SDK.
+Ideally we would like you to be able to generate the XML without ever needing to look at the XSD. 
+For this reason we now provide the `xsdSequenceOrder` in the `fields.json`.
+One possible algorithm to use for sorting notice XML will be provided in the editor demo project, the previous algorithm was fully XSD sequence based.
```

Comment:
Instead of referring to future example, we should mention what the information that this property gives. See its description in https://docs.ted.europa.eu/eforms/latest/fields/index.html

Reply:
The previous editor demo sorting algorithm that was fully XSD based will work with SDK 1.7 as it does not use the xsdSequenceOrder so the older text is correct in this regard. It is confusing that we provided incomplete data.
Unless we do a big patching effort the data and XSD of 1.7 this not going to work for 1.7.
The XSDs and also some nodes/fields, NTDs had to be adapted in 1.8 for this to work using only the xsdSequenceOrder for sorting. This is why this targets 1.8.

Reply:
Like bertrand suggests you need to include the link. However, get rid of the text that does not say anything in particular and think about this as what it is: a "guide". You have a developer in front of you reading this and you are guiding him/her. They do not expect you to telthem RTFM go figure it out in that link. They expect to be guided. So tell them the basics:
Every field (that is not pointing to an attribute) and every node in fields json have this new property. An this property contains an array of xml elements. And for every element we give you an integer number which you can use to sort the array of elements. And that this order that we provide is the one that should be used when generating the xml to avoid xsd valiadtion issues. And, like we said, provide a link to the field metadata documentation page. You can add the link to the first reference of [xsdSequenceOrder](https://docs.ted.europa.eu/eforms/latest/fields/index.html).